src/scorer.py

```diff
@@ -125,28 +128,60 @@ def diff_insert(line: str) -> None:
         def diff_delete(line: str) -> None:
             deletions.append(line)
 
-        self.differ.set_seq1([line.mnemonic for line in cand_seq])
-        result_diff = self.differ.get_opcodes()
+        if self.algorithm == "levenshtein":
+            import Levenshtein
+
+            # I wish I understood what this function actually did
+            # My loose understanding is thus:
+            #     The function converts a list of mnemonics into a single string,
+            #     but the "characters" of the string are not ascii values?
```

Comment:
This is exactly correct. The Levenshtein library unfortunately only allows performing diffs on char sequences while we want to diff string sequences, so we map one to the other. As long as we don't exceed 0x110000 different symbols this hack works out...

I suggest removing/rewording this comment because it doesn't quite fit in the code base :) (It's useful context for the PR though. In the past I've done a self-review of the PR to similar effect.)